src/test/java/com/iexec/commons/containers/client/DockerClientInstanceTests.java

```diff
@@ -404,6 +402,7 @@ void shouldRunSuccessfullyAndWaitForContainerToFinish() throws TimeoutException
         assertThat(dockerRunResponse.getContainerExitCode()).isZero();
         assertThat(dockerRunResponse.getStdout().trim()).isEqualTo(msg);
         assertThat(dockerRunResponse.getStderr()).isEmpty();
+        assertThat(dockerRunResponse.getExecutionDuration()).isGreaterThan(Duration.of(2, ChronoUnit.SECONDS));
```

Comment:
Do we prefer this or the following ?
```suggestion
        assertThat(dockerRunResponse.getExecutionDuration()).isGreaterThan(Duration.ofSeconds(2));
```